src/main/java/io/xcherry/core/rpc/RPC.java

```diff
@@ -8,6 +9,18 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.METHOD)
 public @interface RPC {
+    /**
+     * RPC methods will time out after the specified number of seconds. Set to 0 to disable timeouts.
+     *
+     * @return the timeout duration in seconds.
+     */
     int timeoutInSeconds() default 0;
-// TODO: persistence
+
+    /**
+     * Use the {@link RpcPersistenceReadRequest} with the same name to load persistence in RPC methods.
+     * Load by default as specified in {@link PersistenceSchema}.
+     *
+     * @return the name as defined in a {@link RpcPersistenceReadRequest}.
+     */
+    String rpcPersistenceReadRequestName() default "";
```

Comment:
Just a note that we need another PR to clean up things a little big to get rid of App database later and move back to a simpler model like iwf..